### ISSUE TITLE
wifi icon looks like energy on manjaro

### ISSUE THREAD

**Issue by Askmewho** [grassmunk/Chicago95#109]
dude... amazing worke here.

**Comment by EMH-Mark-I** [comment]
Thank-you.

Are you reporting an issue? You say that the Wifi icon looks like energy? It should appear as this 

![image](https://user-images.githubusercontent.com/28378002/75600384-595a0780-5a74-11ea-8907-9cffdc6a0044.png) 
![image](https://user-images.githubusercontent.com/28378002/75600389-6f67c800-5a74-11ea-9778-2583a1f89b96.png) 
![image](https://user-images.githubusercontent.com/28378002/75600391-75f63f80-5a74-11ea-82a2-b1eb3eaacc17.png) 
![image](https://user-images.githubusercontent.com/28378002/75600394-7d1d4d80-5a74-11ea-8aa6-c66d9eda90bb.png) 
![image](https://user-images.githubusercontent.com/28378002/75600398-827a9800-5a74-11ea-8d43-3376cc8653f9.png)

**Comment by Askmewho** [comment]
Thanks, yeah. Maybe that's the wifi icon (i didnt have wifi on my w95 pc hahah) . Did you have any intention to port it to other platforms, like lxde or pixel? I do love your work, I've mention your work on a video (I have an small channel) running your them on manjaro on a raspberry pi 4

**Comment by EMH-Mark-I** [comment]
The icon theme derives from Classic95 with additions, but I'm not sure if there was a wireless network icon for Windows PC's during that time. (I hadn't ever used a computer with a wireless card until sometime around mid 00's.) It's probably an icon that was used for battery power or something else, but has been repurposed here for the wireless icon.

I have added support for a select few other desktops; MATE and Cinnamon. Mate should work across platforms that support GTK3.22 and 3.24, however Cinnamon I'm not so sure. I created the Cinnamon portion of the theme from a desktop running GTK3.24.

As for LXDE, I'm not sure. I thought they moved to Qt, and are now called LXQt? I haven't focused much on Qt desktops; KDE being the other. Grassmunk and a few others were looking at it at one point. My focus has primarily been on the GTK aspect and beating it into shape.
